plotting/README.md

```diff
@@ -0,0 +1,32 @@
+# Autoreduction Plotting
+
+This document provides documentation for the plotting package as part of the Autoreduction project.
```

Comment:
~This document provides d~Documentation for the plotting package as part of the Autoreduction project.